benchmarks/ShuffleBench.cu

```diff
@@ -53,28 +57,30 @@ static void sortArgsGenerator( benchmark::internal::Benchmark* b )
 }
 
 
-BENCHMARK_TEMPLATE( benchmarkFunction, FeistelBijectiveShuffle<thrust::device_vector<uint8_t>> )->Apply( argsGenerator );
-BENCHMARK_TEMPLATE( benchmarkFunction, FeistelBijectiveSortShuffle<thrust::device_vector<uint8_t>> )->Apply( sortArgsGenerator );
-BENCHMARK_TEMPLATE( benchmarkFunction, FeistelBijectiveScanShuffle<thrust::device_vector<uint8_t>> )->Apply( sortArgsGenerator );
-BENCHMARK_TEMPLATE( benchmarkFunction, SPNetworkBijectiveShuffle<thrust::device_vector<uint8_t>> )->Apply( argsGenerator );
-BENCHMARK_TEMPLATE( benchmarkFunction, SPNetworkBijectiveSortShuffle<thrust::device_vector<uint8_t>> )
-    ->Apply( sortArgsGenerator );
-BENCHMARK_TEMPLATE( benchmarkFunction, SPNetworkBijectiveScanShuffle<thrust::device_vector<uint8_t>> )
+BENCHMARK_TEMPLATE( benchmarkFunction, FeistelBijectiveShuffle<thrust::device_vector<uint64_t>> )->Apply( argsGenerator );
```

Comment:
I used uint8_t so we could do larger counts without running out of memory. Was there a particular need to switch

Reply:
Nobody actually shuffles 8 bit keys, a more realistic benchmark will use 64 bit keys, typically integer positions of items, like in cross validation.

Reply:
Fair enough